alacritty_terminal/src/term/cell.rs

```diff
@@ -184,11 +211,22 @@ impl LineLength for grid::Row<Cell> {
 
 #[cfg(test)]
 mod tests {
+    use std::mem;
+
     use super::{Cell, LineLength};
 
     use crate::grid::Row;
     use crate::index::Column;
 
+    #[test]
+    fn cell_size_is_below_cap() {
+        // Estimated cell size on 64-bit architectures.
+        const ESTIMATED_CELL_SIZE: usize = 24;
```

Comment:
```suggestion
        // Expected cell size on 64-bit architectures.
        const EXPECTED_MAX_CELL_SIZE: usize = 24;
```

I think `expected` is a bit better than `estimated` here. We aren't really guessing at a value, but we **know** it should be under this exact number.